### PR TITLE
use prebuild libraries for openssl.

### PR DESCRIPTION
**Pull request by blagoev**
use prebuild libs for openssl to not build openssl locally

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,12 +19,12 @@ RUN yum -y install \
  && yum clean all \
  \
   # TODO: install openssl in /usr/local
- && curl -SL https://www.openssl.org/source/openssl-1.1.1b.tar.gz | tar -zxC / \
- && cd /openssl-1.1.1b \
- && ./config --prefix=/usr \
- && make && make install \
- && rm -rf /openssl-1.1.1b \
- && cd /tmp \
+#  && curl -SL https://www.openssl.org/source/openssl-1.1.1b.tar.gz | tar -zxC / \
```
Comment (kneth) [inline Dockerfile:22]: I guess we can remove this part?

File: scripts/download-realm.js [hunk 2]
```diff
@@ -164,6 +167,7 @@ function getCoreRequirements(dependencies, options, required = {}) {
         case 'linux':
             required.CORE_SERVER_FOLDER += `/linux/${flavor}`;
             required.CORE_ARCHIVE = `realm-core-${flavor}-v${dependencies.REALM_CORE_VERSION}-Linux-devel.tar.gz`;
+            required.openssl = "https://static.realm.io/downloads/openssl/1.1.1b/Linux/x86_64/openssl.tgz";
```
Comment (kneth) [inline scripts/download-realm.js:170]: Maybe `openssl` -> `OPENSSL` to be consistent with the rest.

Comment (blagoev) [reply scripts/download-realm.js]: 👍

File: Dockerfile (hunk 1 above)
Comment (blagoev) [reply Dockerfile]: 👍

**Pull request merged**